cdapython/utils/utility.py

```diff
@@ -111,20 +111,21 @@ def unique_terms(
     verbose: bool = True,
     limit: int = 100,
 ) -> Paged_Result:
+    # TODO : Should the value be changed? The "show_term_count" parameter was copied from Swagger-generated code. Is this description sufficient?
     """
     Show all unique terms for a given column.
     Args:
-        col_name (str): _description_
-        system (str, optional): _description_. Defaults to "".
-        offset (int, optional): _description_. Defaults to 0.
-        host (Optional[str], optional): _description_. Defaults to None.
-        verify (Optional[bool], optional): _description_. Defaults to None.
-        async_req (Optional[bool], optional): _description_. Defaults to True.
-        show_sql (bool, optional): _description_. Defaults to False.
-        show_counts (bool, optional): _description_. Defaults to False.
-        show_term_count (bool, optional): _description_. Defaults to False.
-        verbose (bool, optional): _description_. Defaults to True.
-        limit (int, optional): _description_. Defaults to 100.
+        col_name (str): This is the default way to search for a unique term from the CDA service API.
+        system (str, optional): his is an optional parameter used to filter the search values by data center, such as "GDC", "IDC", "PDC", or "CDS". Defaults to "".
+        offset (int, optional): The number of entries to skip. Defaults to 0.
+        host (Optional[str], optional): This is where the user can set a host for a different server. Defaults to None.
+        verify (Optional[bool], optional): This will send a request to the cda server without verifying the SSL Cert Verification. Defaults to None.
+        async_req (Optional[bool], optional): Execute request asynchronously. Defaults to True.
+        show_sql (bool, optional): This will show the sql returned from the server. Defaults to False.
+        show_counts (bool, optional): Deprecated will be removed: This will show or hide the count. Defaults to False.
```

Comment:
Is this really deprecated or nonfunctional?

Reply:
If it's nonfunctional in the current state of the code and needs to be functional - state that.  If it's functional and going away, deprecated is the correct term.

Reply:
The show_counts param currently nonfunctional. it is currently waiting on a response to see if it is needed to re-implement.